pycbc/results/versioning.py

```diff
@@ -92,9 +92,11 @@ def add_info_new_version(info_dct, curr_module, extra_str):
 
     pycbcinfo = {}
     pycbcinfo['Name'] = 'PyCBC'
-    pycbcinfo['ID'] = pycbc.version.version
+    pycbcinfo['ID'] = pycbc.version.git_hash
     pycbcinfo['Status'] = pycbc.version.git_status
-    pycbcinfo['Version'] = pycbc.version.release or ''
+    pycbcinfo['Version'] = pycbc.version.version
+    if pycbc.version.release:
+        pycbcinfo['Version'] += ' (release)'
```

Comment:
Do we want the text appended to the version string? Won't that make it an invalid format for release?